docs/packages/pkg/Immer.rst

```diff
@@ -0,0 +1,20 @@
+.. spelling::
+
+    Immer
+
+.. index::
+  single: unsorted ; Immer
+
+.. _pkg.immer:
```

Comment:
I started first with a lower case `i` and noticed later that the developer from the `immer` project use `Immer` with upper case `I` in `find_package()` .... so I changed all to upper case `I` but not the branches .... this line 8 here in the docu ... should this also an upper case `I` like `.. _pkg.Immer` ?

Reply:
This should be capital, it’s not related to a branch

Reply:
ok done.